force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +279,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book myBook = new Book('A Brief History of Time', 'Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String myBookDetails = mybook.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return myBookDetails; // Replace null with the variable you used to store the result
```

Comment:
As can example of how you chain these together. Your method returns a String. So  you can create the Book and getBookDetails all in one shot. You are not using the object for anything else. 

 return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();